OpenRA.Game/Game.cs

```diff
@@ -314,10 +315,16 @@ internal static void Initialize(Arguments args)
 			GlobalChat = new GlobalChat();
 
 			Mods = new InstalledMods(customModPath);
-			Console.WriteLine("Available mods:");
+			Console.WriteLine("Internal mods:");
 			foreach (var mod in Mods)
 				Console.WriteLine("\t{0}: {1} ({2})", mod.Key, mod.Value.Metadata.Title, mod.Value.Metadata.Version);
 
+			var engine = args.GetValue("Exec.Engine", Assembly.GetEntryAssembly().Location);
```

Comment:
I'm not sure if `Exec.` is the best prefix for this, so suggestions are welcome.  I wanted to avoid `Launch.` because the rest of the launch args aren't parsed until much later in the mod loading.  Future prs will be adding `.DefaultMod` and `.ModsDirectory` arguments on this same prefix.